src/hooks/mention.test.ts

```diff
@@ -1,7 +1,9 @@
 /* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Can this be removed ? Maybe we need to change the eslint rule to take into account the devDependencies ?